modules/apps/journal/journal-web/src/main/resources/META-INF/resources/js/JournalPortlet.es.js

```diff
@@ -137,6 +139,17 @@ class JournalPortlet extends PortletBase {
 		this._saveArticle(actionName);
 	}
 
+	/**
+	 * @private
+	 */
+	_onLocaleChanged(context, languageId) {
+		const {availableLocales} = context;
```

Comment:
Could we destructure the `availableLocales` on the function call? `_onLocaleChanged({availableLocales}, languageId)`

Reply:
@interaminense, you are right!!! I'll make this change and resend. Thank you!